release-notes/3.0/preview/api-diff/preview7/3.0-preview7_System.md

```diff
@@ -0,0 +1,33 @@
+# System
+
+``` diff
+ namespace System {
+     public static class MemoryExtensions {
+-        public static ReadOnlySpan<T> Trim<T>(this ReadOnlySpan<T> memory, ReadOnlySpan<T> trimElements) where T : IEquatable<T>;
```

Comment:
These changes to MemoryExtensions LGTM (from https://github.com/dotnet/corefx/pull/38287).
cc @grant-d